projects/admin/src/app/frontpage/frontpage.component.ts

```diff
@@ -44,7 +44,7 @@ export class FrontpageComponent implements OnInit {
   ngOnInit() {
     this.initLinksMenu();
     this._localStorageService.onSet$.subscribe(() => {
-      const link = this.linksMenu.entries[3].entries.find(
+      const link = this.linksMenu.entries[4].entries.find(
```

Comment:
It would be great to find the my library link instead of hard code the index. This can be done later but please add a TODO here.

Reply:
Ok Done. Find method implemented.